### PR TITLE
Add workaround for M4A cover recognized as 'uint8'

### PR DESCRIPTION
**Pull request by pioug**
Cf Issue https://github.com/aadsm/jsmediatags/issues/26.

### CODE REVIEW

File: src/MP4TagReader.js [hunk 1]
```diff
@@ -208,6 +208,10 @@ class MP4TagReader extends MediaTagReader {
       var dataLength = atomSize - atomHeader;
       var atomData;
 
+      if (atomName === 'covr' && type === 'uint8') {
```
Comment (aadsm) [inline src/MP4TagReader.js]: Can you add a comment here explaining why we're doing this hack?

PS: thank you for the PR and sorry for taking so long to look into it.

**Comment by aadsm** [comment]
Could you also add a unit test for this? It should be added in https://github.com/aadsm/jsmediatags/blob/master/src/__tests__/MP4TagReader-test.js

**Comment by pioug** [comment]
Thank you for your feedback. I added the unit test, not sure if it is the correct way though!

File: src/__tests__/MP4TagReader-test.js [hunk 2]
```diff
@@ -167,3 +167,31 @@ describe("MP4TagReader", function() {
     });
   });
 });
+
+describe("MP4TagReader", function() {
```
Comment (aadsm) [inline src/__tests__/MP4TagReader-test.js]: No need to repeat this, just add your new `pit` function call after line 196.
And put your custom mp4FileContents/mediaFileReader/tagReader variable logic inside the `pit` too.

**Comment by pioug** [comment]
Thanks for reviewing, I updated the PR

**Pull request merged**